src/applications/proxy-rewrite/proxy-rewrite-entry.jsx

```diff
@@ -137,7 +138,21 @@ function activateInjectedAssets() {
 
   document.addEventListener('DOMContentLoaded', _e => {
     activateHeaderFooter(observer);
-    mountReactComponents(createCommonStore());
+    fetch(`${environment.BASE_URL}/generated/headerFooter.json`)
```

Comment:
Not sure if this will work correctly on live Teamsite pages?

Reply:
I think it will, but will we need `{ cors: true }` here? 

I'm probably just overlooking it, but where is `headerFooter.json` being written?

Reply:
Nope, good catch, I lost the code that wrote that file somewhere along the way. I'll add it back.